config/vm.args

```diff
@@ -1,4 +1,4 @@
--sname aeternity@localhost
+-sname epoch@localhost
```

Comment:
Used in:
* [User config discovery](https://github.com/aeternity/aeternity/blob/6248401116910cf053d4d184cb723b6c544ddd0d/apps/aeutils/src/aeu_env.erl#L293).
* [DB](https://github.com/aeternity/aeternity/blob/6248401116910cf053d4d184cb723b6c544ddd0d/apps/aecore/src/aec_db.erl).